http2_general/test_h2_sticky_scheduler.py

```diff
@@ -0,0 +1,164 @@
+"""Test module for http2 and Sticky Cookie Scheduler."""
+import http
+
+from framework import tester
+from helpers.response_parser import parse_response
+
+
+class H2StickySchedulerTestCase(tester.TempestaTest):
+    """Sticky Cookie H2 test case."""
+
+    clients = [
+        {
+            'id': 'curl-init',
+            'type': 'external',
+            'binary': 'curl',
+            'ssl': True,
+            'cmd_args': '-Ikf -v --http2 https://good.com/',
+        },
+        {
+            'id': 'curl-good',
+            'type': 'external',
+            'binary': 'curl',
+            'ssl': True,
+            'cmd_args': '-Ikf -v --http2 https://good.com/ -H "Cookie: BIGipServerwebapps-sea-7775=1695682732.24350.0000"',  # noqa:E501
+        },
+        {
+            'id': 'curl-bad',
+            'type': 'external',
+            'binary': 'curl',
+            'ssl': True,
+            'cmd_args': '-Ikf -v --http2 https://bad.com/ -H "Cookie: BIGipServerwebapps-sea-7775=1695682732.24350.0000"',  # noqa:E501
+        },
+    ]
+
+    tempesta = {
+        'config': """
+            listen 127.0.0.4:8765 proto=h2;
+
+            srv_group default {
+                server ${server_ip}:8000;
+            }
+            vhost default {
+                proxy_pass default;
+            }
+            vhost good.com {
+                proxy_pass default;
+                sticky {
+                    sticky_sessions;
+                    cookie enforce;
+                    secret "f00)9eR59*_/22";
+                }
+            }
+            tls_match_any_server_name;
+            tls_certificate ${tempesta_workdir}/tempesta.crt;
+            tls_certificate_key ${tempesta_workdir}/tempesta.key;
+            cache 0;
+            cache_fulfill * *;
+            block_action attack reply;
+            http_chain {
+                host == "bad.com"	-> block;
+                host == "good.com" -> default;
+            }
```

Comment:
Even, if we will have `Host` header with `good.com` value, request doesn't hit `good.com` vhost, because it routes to `default`. Also, please rename `default` vhost to proper use of `tls_match_any_server_name`